src/options/options.html

```diff
@@ -54,59 +54,63 @@
 		<form>
 			<section>
 				<ul>
-					<h1 data-i18n="">Unicode autocorrection</h1>
-					<span class="helper-text">
+					<h1 data-i18n="__MSG_titleUnicodeAutocorrection__">Unicode autocorrection</h1>
+					<span class="helper-text" data-i18n="__MSG_optionHelperTextAutocorrection__"  data-opt-i18n-keep-children>
 					You can configure the automatic corrections that are done when you are typing text here.<br>
 					All the autocorrections are done after typing the first nonmatching character following the character sequence, such as a space (␣). Press Backspace (⌫) to undo an autocorrection.
-					Only the quotes replacement is done immediately.<br>
-					For Emoji autocorrection, including <code>:colon:</code> shortcodes and Emoticons, please try our <a href="https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/">🤩&nbsp;Awesome Emoji Picker</a> add-on.</span>
+					Only the quotes replacement is done immediately.
+				</span><br>
+					<span class="helper-text" data-i18n="__MSG_optionHelperTextAutocorrectionEmoji__"  data-opt-i18n-keep-children>
+					For Emoji autocorrection, including <code data-i18n="__MSG_optionHelperTextAutocorrectionEmojiColonsCode__">:colon:</code> shortcodes and Emoticons, please also try out our <a href="https://addons.mozilla.org/firefox/addon/awesome-emoji-picker/" data-i18n="__MSG_optionHelperTextAutocorrectionEmojiLinkText__"  data-i18n-href="__MSG_optionHelperTextAutocorrectionEmojiLink__">🤩&nbsp;Awesome Emoji Picker</a> add-on.
```

Comment:
While it does not effect this PR, note that the AMO URL will need to be updated after I create the Thunderbird and Chrome PRs for the Awesome Emoji Picker...

Reply:
Ah indeed, at best dynamically. Yeah, thanks for the notice.

Reply:
No problem.